_translations/de.yml

```diff
@@ -575,33 +733,43 @@ de:
     mining: "Mining"
-    miningtxt: "Sie können damit beginnen <a href=\"http://www.bitcoinmining.com/\">Bitcoins zu minen</a>, um bei der Verarbeitung von Transaktionen zu helfen. Um das Netzwerk zu schützen, sollten Sie kleineren Mining Pools beitreten, und dezentrale Pools wie <a href=\"http://p2pool.in/\">P2Pool</a> oder <a href=\"https://en.bitcoin.it/wiki/Comparison_of_mining_pools\">Pools</a> mit Unterstützung von getblocktemplate (GBT), bevorzugen."
+    miningtxt: "Sie können damit beginnen, <a href=\"http://www.bitcoinmining.com/\">Bitcoins zu minen</a>, um bei der Verarbeitung von Transaktionen zu helfen. Um das Netzwerk zu schützen, sollten Sie <a href=\"http://mempool.info/pools\">kleineren Mining-Pools</a> beitreten, und dabei dezentrale Pools wie <a href=\"http://p2pool.in/\">P2Pool</a> oder <a href=\"https://en.bitcoin.it/wiki/Comparison_of_mining_pools\">Pools</a>, die getblocktemplate (GBT) unterstützen, bevorzugen."
     develop: "Entwicklung"
-    developtxt: "Bitcoin ist quelloffen. Wenn Sie ein Entwickler sind, können Sie Ihre Superkräfte nutzen um Gutes zu tun und <a href=\"#development#\">Bitcoin zu verbessern</a>. Sie können auch beeindruckende neue Dienste oder Software bauen, die Bitcoin verwenden."
+    developtxt: "Bitcoin ist open-source. Wenn Sie ein Entwickler sind, können Sie Ihre Superkräfte nutzen, um Gutes zu tun und <a href=\"#development#\">Bitcoin zu verbessern</a>. Sie können auch beeindruckende neue Dienste oder Software bauen, die Bitcoin verwenden."
     donation: "Spenden"
-    donationtxt: "Der einfachste Weg zu helfen ist ein paar Bitcoins an die Bitcoin Foundation zu <a href=\"https://bitcoinfoundation.org/about/donate/\">spenden</a>. Oder SIe können bei der Finanzierung von in der nahen Zukunft möglicherweise nützlichen Projekten helfen, die mit Bitcoin im Zusammenhang stehen."
+    donationtxt: "Der einfachste Weg, um zu helfen, ist ein paar Bitcoins an BitGive zu <a href=\"http://bitgivefoundation.org/donate-now/\">spenden</a>. Sie können aber auch bei der Finanzierung von Projekten helfen, die mit Bitcoin im Zusammenhang stehen und von denen Sie glauben, dass sie in Zukunft nützlich sind."
     nonprofit: "Organisationen"
-    nonprofittxt: "Die <a href=\"https://bitcoinfoundation.org/\">Bitcoin Foundation</a> und viele andere <a href=\"#community##[community.non-profit]\">gemeinnützige Organisationen</a> sind dem Schutz und der Förderung von Bitcoin gewidmet. Sie können diesen Gruppen helfen in dem Sie Ihnen beitreten und an den Projekten, Diskussionen und Ereignissen teilhaben."
+    nonprofittxt: "Viele </a>gemeinnützige Organisationen</a> sind dem Schutz und der Förderung von Bitcoin gewidmet. Sie können diesen Gruppen helfen, indem Sie diesen beitreten und an Projekten, Diskussionen und Events mitwirken."
```

Comment:
`</a>` should be removed there (in Transifex @Komodorpudel) then pushed here.

Reply:
I managed to find the string ☝️ on Transifex here: https://www.transifex.com/bitcoinorg/bitcoinorg/viewstrings/#de/bitcoinorg/20474254?key=support-bitcoin.nonprofittx

Reply:
Hey Jacob,

Thank you very much!
I went through all the warnings and corrected every faulty link I could found.

I hope that there are no mistakes left now 😊